lib/node_modules/@stdlib/blas/scopy/lib/main.js

```diff
@@ -0,0 +1,63 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var base = require( '@stdlib/blas/base/scopy' ).ndarray;
+var factory = require( '@stdlib/blas/tools/swap-factory' );
```

Comment:
@aman-095 I understand why you are attempting to reuse `swap-factory`; however, I'd advise not doing that, as the behavior is not guaranteed to be the same, as mentioned in the test file below.

Reply:
You need to actually create `tools/copy-factory`.

Reply:
You can open as a separate PR.